pkg/defaults/categories/categories.go

```diff
@@ -55,7 +55,7 @@ func readCategoryFile(path string) (*storage.PolicyCategory, error) {
 	utils.CrashOnError(err)
 
 	var category storage.PolicyCategory
-	err = jsonpb.Unmarshal(bytes.NewReader(contents), &category)
+	err = jsonutil.JSONUnmarshaler().Unmarshal(bytes.NewReader(contents), &category)
```

Comment:
Overall looks ok, but feels that `jsonutil.JSONUnmarshaler().Unmarshal(` could be made shorter, e.g. `jsonutil.JSONBytesToProto(` - a new wrapper that covers cases in this PR.
Feel free to treat it as optional.

Reply:
I like that suggestion.  That will make things smoother as we work through the rest of the tickets.